Beta_Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.py

```diff
@@ -0,0 +1,328 @@
+    return_outputs(human_readable, ec)
+
+
+def get_analyzed_resource(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    kwargs = {
+        'analyzerArn': args.get('analyzerArn'),
+        'resourceArn': args.get('resourceArn')
+    }
+
+    response = client.get_analyzed_resource(**kwargs)
+    data = json.loads(json.dumps(response['resource'], cls=DatetimeEncoder))
+    data['analyzerArn'] = args.get('analyzerArn')
+
+    ec = {'AWS.AccessAnalyzer.Analyzers(val.analyzerArn === obj.analyzerArn).Resource(val.resourceArn === obj.resourceArn)': data}
+    human_readable = tableToMarkdown("AWS Access Analyzer Resource", data)
+    return_outputs(human_readable, ec)
+
+
+def get_finding(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    kwargs = {
+        'analyzerArn': args.get('analyzerArn'),
+        'id': args.get('findingId')
+    }
+
+    response = client.get_finding(**kwargs)
+    data = json.loads(json.dumps(response['finding'], cls=DatetimeEncoder))
+    data['analyzerArn'] = args.get('analyzerArn')
+
+    ec = {'AWS.AccessAnalyzer.Analyzers(val.analyzerArn === obj.analyzerArn).Findings(val.resourceArn === obj.resourceArn)': data}
+    human_readable = tableToMarkdown("AWS Access Analyzer Resource", data)
+    return_outputs(human_readable, ec)
+
+
+def start_resource_scan(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    kwargs = {
+        'analyzerArn': args.get('analyzerArn'),
+        'resourceArn': args.get('resourceArn')
+    }
+
+    client.start_resource_scan(**kwargs)
+    return_outputs("Resource scan request sent.")
+
+
+def update_findings(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+
+    ids = args.get('findingIds').split(',')
+
+    kwargs = {
+        'analyzerArn': args.get('analyzerArn'),
+        'ids': ids,
+        'status': args.get('status')
+    }
+
+    client.update_findings(**kwargs)
+    return_outputs("Findings updated")
+
+
+def test_function():
+    try:
+        client = aws_session()
+        response = client.list_analyzers()
+        if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+            return_outputs('ok')
+
+    except Exception as e:
+        return return_error(str(e))
+
+
+"""EXECUTION BLOCK"""
+try:
```

Comment:
a general comment:
please change the command functions names that are called under this to `<name>_command()`